espnet2/enh/decoder/stft_decoder.py

```diff
@@ -59,3 +73,60 @@ def forward(self, input: ComplexTensor, ilens: torch.Tensor):
             wav = wav.reshape(bs, -1, wav.size(1)).transpose(1, 2)
 
         return wav, wav_lens
+
+    def _get_window_func(self):
+        window_func = getattr(torch, f"{self.window}_window")
+        window = window_func(self.win_length)
+        n_pad_left = (self.n_fft - window.shape[0]) // 2
+        n_pad_right = self.n_fft - window.shape[0] - n_pad_left
+        window = torch.cat(
+            [torch.zeros(n_pad_left), window, torch.zeros(n_pad_right)], 0
+        )
+        return window
```

Comment:
This function seems unused in `forward_streaming`, which I think should be used.

Reply:
solved.